.yarnrc.yml

```diff
@@ -4,4 +4,4 @@ defaultSemverRangePrefix: ""
 
 nodeLinker: node-modules
 
-yarnPath: .yarn/releases/yarn-3.2.1.cjs
+yarnPath: .yarn/releases/yarn-3.3.0.cjs
```

Comment:
It's good practice to avoid mixing dependencies updates and feature implementations, except if it is required by the new implementation. It makes it easier to roll back separate concerns if needed. Though it should be fine for this time.